02-Style-Guide.md

```diff
@@ -24,9 +27,25 @@ Other style guidelines
 - Use override (and not virtual) when overriding methods through inheritance. Obviously the original method still needs virtual (and not override).
 - Use `#pragma once`. It’s supported by every modern compiler, and is simpler to use than `#ifdef` guards.
 - Avoid casts. If you have to cast, use C++ casts (static_cast, dynamic_cast, const_cast, reinterpret_cast, preferred in that order) and cry a little bit inside.
-- Header file inclusion<sup>[1](http://gamesfromwithin.com/physical-structure-and-c-part-1-a-first-look)</sup>:
-  - Every cpp file includes its own header file first.
-  - A header file must include all the header files necessary to parse it.
-  - A header file should have the bare minimum number of header files necessary to parse it. Prefer forward-declarations when possible, i.e. whenever you only need pointers/references to a class outside the std namespace<sup>[2](https://groups.google.com/forum/#!topic/comp.lang.c++.moderated/XBKFjGNquDM)</sup>.
 
-TODO: Still under development.
+## Header Files
+
+Header file inclusion<sup>[1](http://gamesfromwithin.com/physical-structure-and-c-part-1-a-first-look)</sup>:
+- Every cpp file includes its own header file first.
+- A header file must include all the header files necessary to parse it.
+- A header file should have the bare minimum number of header files necessary to parse it. Prefer forward-declarations when possible, i.e. whenever you only need pointers/references to a class outside the std namespace<sup>[2](https://groups.google.com/forum/#!topic/comp.lang.c++.moderated/XBKFjGNquDM)</sup>.
+
+## Namespaces
+
+Never put `using namespace x` in a header file. If you want a short-hand in a header file, use aliases (i.e. `namespace x = long_namespace_x`) inside of a namespace declared in that header file, as in
```

Comment:
No, I don't have another reason. I think it would only seem iffy to me if done outside the namespace.

My experience talking to a lot of people lately who are completely new to C/C++ is that the idea that literally everything in the header also happens everywhere it's included is unintuitive, which is why I lean towards pointing that out more often.